src/drivers/imx/sai.c

```diff
@@ -192,14 +199,14 @@ static inline int sai_set_config(struct dai *dai,
 	/* DAI clock master masks */
 	switch (config->format & SOF_DAI_FMT_MASTER_MASK) {
 	case SOF_DAI_FMT_CBS_CFS:
-		trace_sai("SAI: codec is slave");
+		dai_info(dai, "SAI: codec is slave");
```

Comment:
I think this one and the next one shall be moved to dai_dbg.

Reply:
@paulstelian97 I do think that this PR should keep the existing behavior and we can do enhancements later.

Reply:
Fair point.